lib/assets/javascripts/cdb

```diff
@@ -1 +1 @@
-Subproject commit a2b8834622f2b4d3b849a7e810cff998d982844d
+Subproject commit 5bee7c8c6c6b2a62ec29c598f9c4331a9cb831fb
```

Comment:
You'll want to make sure that your git work tree is clean (no diffs if you `git status`), and then do a `git submodule sync && git submodule update`. We can take a look in the morning if that doesn't cut it.